src/stars.css

```
@@ -0,0 +1,88 @@
+.stars {
+  display: flex;
+  width: 96px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(n) {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+}
```

A lot of repeating styles, just have these common styles in stars_star and don't repeat them below
```suggestion
.stars__star {
  background-repeat: no-repeat;
  background-position: center;
  background-image: url(./images/star.svg);
  height: 16px;
  width: 16px;
}
```